skt/mls.py

```diff
@@ -322,3 +323,31 @@ def get_meta_table_item(meta_table: str, item_name: str, aws_env: AWSENV = AWSEN
         raise MLSModelError(response.get("error"))
     else:
         return results
+
+
+def meta_to_pandas(meta_table: str, aws_env: AWSENV = AWSENV.STG.value) -> Any:
+    """
+    Get a meta_table as pandas dataframe
+    Args. :
+        - meta_table   :   (str) the name of meta_table
+        - aws_env      :   (str) AWS ENV in 'stg / prd / dev' (default is 'stg')
+    Returns :
+        - A Pandas dataframe type of the item_meta
+    """
+    assert type(meta_table) == str
+    assert type(aws_env) == str
+
+    url = AB_URL_PREFIX
+    if aws_env in (AWSENV.STG.value, AWSENV.DEV.value):
+        url = f"{url}.{aws_env}"
+    url = f"{url}.sktmls.com{MLS_META_API_URL}/{meta_table}"
+
+    response = requests.get(url).json()
+
```

Comment:
meta_table이 존재하지 않는 경우 예외처리 추가해주세요

```suggestion
    response = requests.get(url).json()

```